ietf-microwave-radio-link.yang

```diff
@@ -60,13 +61,30 @@ module ietf-microwave-radio-link {
   // replace the revision date with the module publication date
   // the format is (year-month-day)
 
-  revision 2024-09-11 {
+  revision 2024-03-03 {
     description
-      "Draft - Issue #35 (grouping common-microwave-properties
-               Issue #31 added minimum-nominal-power";
+      "pretty printing";
     reference
       "RFC XXX: A YANG Data Model for Microwave Radio Link";
   }
+  revision 2023-10-06 {
+    description
+      "To add the summary of the changes in this revision:
+
+       - more updates related to git Issue #13 and #16.
+
+       - git Issue #13 and #16 and discussion updates for acm-profile
+
+       - Updated with proposal for BCA.";
+    reference
+      "RFC XXX: A YANG Data Model for Microwave Radio Link";
+  }
```

Comment:
See: https://datatracker.ietf.org/doc/html/draft-ietf-netmod-rfc8407bis#name-module-header-meta-and-revi

> For an unpublished module, a complete history of each unpublished module revision is not required. That is, within a sequence of draft versions, only the most recent revision need be recorded in the module.

```suggestion
      "To add the summary of the changes in this revision:

       - more updates related to git Issue #13 and #16.

       - git Issue #13 and #16 and discussion updates for acm-profile

       - Updated with proposal for BCA.";
    reference
      "RFC XXX: A YANG Data Model for Microwave Radio Link";
  }
```